pkg/operator/controller/names.go

```diff
@@ -95,11 +95,11 @@ func RsyslogConfigMapName(ic *operatorv1.IngressController) types.NamespacedName
 
 // RouterEffectiveDefaultCertificateSecretName returns the namespaced name for
 // the in-use router default certificate secret.
-func RouterEffectiveDefaultCertificateSecretName(ci *operatorv1.IngressController, namespace string) types.NamespacedName {
-	if cert := ci.Spec.DefaultCertificate; cert != nil {
-		return types.NamespacedName{Namespace: namespace, Name: cert.Name}
+func RouterEffectiveDefaultCertificateSecretName(ic *operatorv1.IngressController) types.NamespacedName {
+	if cert := ic.Spec.DefaultCertificate; cert != nil {
+		return types.NamespacedName{Namespace: ic.Namespace, Name: cert.Name}
```

Comment:
Whoops, this is wrong. I think hard-coding the namespaces in this file would actually be more clear at the moment...

Reply:
Fixed